### PR TITLE
Add <meta description> to index.html.

### PR DESCRIPTION
**Pull request by 0xbiel**
Added just for index.html. Do you want me to add for the other pages as well?

### CODE REVIEW

**Comment by 0xbiel** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

**Comment by 0xbiel** [comment]
**recheck**

File: website/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <head>
     <title>quick-lint-js finds JavaScript bugs</title>
     <meta charset="utf-8" />
+		<meta name="description" content="quick-lint-js gives you instant feedback as you code. Find bugs in your JavaScript before your finger leaves the keyboard."/>
```
Comment (strager) [inline website/index.html:8]: Nit: Run `yarn fmt` to reformat the code. (It looks like the indentation is way off, for example.)

**Review by strager — APPROVED** [review]
> Do you want me to add for the other pages as well?

Yes!

File: website/install.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <head>
     <title>quick-lint-js installation</title>
     <meta charset="utf-8" />
+    <meta name="description" content="Install quick-lint-js on Linux, MacOS, Windows, VS Code and many more!"/>
```
Comment (strager) [inline website/install.html:8]: Nit: It's spelled macOS, not MacOS.

File: website/license.html [hunk 3]
```diff
@@ -6,6 +6,7 @@
   <head>
     <title>quick-lint-js license information</title>
     <meta charset="utf-8" />
+    <meta name="description" content="quick-lint-js license information."/>
```
Comment (strager) [inline website/license.html:9]: Minor: This description just repeats the title. Is it worth including?

**Review by strager — APPROVED** [review]
Looks great to me. Let me know when I should merge it.

File: website/license.html (hunk 3 above)
Comment (0xbiel) [reply website/license.html:9]: yeah, i don't think so. It's pretty self-explanatory.

**Comment by 0xbiel** [comment]
Removed the <meta description> from license.html and fixed the macOS typo. You can merge now!

**Pull request merged**